app/policies/user_policy.rb

```diff
@@ -33,4 +33,8 @@ def update?
   def destroy?
     @current_user.superadmin? && !@user.superadmin?
   end
+
+  def reset_password?
+    @current_user.superadmin? || @current_user == @user
+  end
```

Comment:
yea, maybe in this case your right, because the update? would still be explicit. I'll change it. 

But in other cases `policy(Xy).destroy?` was used in views for things that where absolutely not related to `destroy?` abilities. 
That is, i reckon, not very explicit.